cdap-app-fabric/src/main/java/io/cdap/cdap/internal/profile/ProfileService.java

```diff
@@ -88,6 +89,23 @@ public Profile getProfile(ProfileId profileId) throws NotFoundException {
     }, NotFoundException.class);
   }
 
+  /**
+   * Get profile information about profiles with a given provisioner name and property value
```

Comment:
Could this be rephrased as "Get list of profiles for a provisioner and its property value from the profile store"?

Reply:
fixed